spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/data/vectorized/parquet/TestParquetDictionaryEncodedVectorizedReads.java

```diff
@@ -60,7 +61,11 @@ public class TestParquetDictionaryEncodedVectorizedReads extends TestParquetVect
 
   @BeforeAll
   public static void startSpark() {
-    spark = SparkSession.builder().master("local[2]").getOrCreate();
```

Comment:
You can't see it here, but testMixedDictionaryNonDictionaryReads fails without the fix included here now that leak detection is added